src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,113 @@
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Text,
+  Menu,
+  MenuButton,
+  MenuList,
+  Badge,
+  HStack,
+} from '@chakra-ui/react';
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import { FilterCheckbox } from '../Catalog/SearchFilter/SearchFilter';
+import { useState, useEffect } from 'react';
+
+const Dropdown = ({ options, filter, selected, defaults, badgeColor, width }) => {
+  const { getCheckboxProps } = filter;
+  const [selectedOptions, setSelectedOptions] = useState([]);
+
+  useEffect(() => {
+    setSelectedOptions(selected);
+  }, [selected]);
+
+  useEffect(() => {
+    // console.log('inital default');
+    if (defaults && defaults[0] !== '')
+      filter.setValue(defaults);
+  }, []);
+
+  useEffect(() => {
+    // console.log('update default', defaults);
+    if (defaults && defaults[0] !== '')
+      filter.setValue(defaults);
+    if (!defaults) {
+      filter.setValue([]);
+    }
+  }, [defaults]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'filter'. Either include it or remove the dependency array.